src/components/ProgramList.tsx

```diff
@@ -11,142 +11,142 @@ import {
 import type { ProgramCardInfo } from "models/Program";
 import { useSession } from "next-auth/client";
 import { programFormat } from "@prisma/client";
-
+import { ProgramCard } from "@components/ProgramCard";
 export const ProgramList: React.FC<{ cardInfo: ProgramCardInfo[] }> = ({
     cardInfo,
 }) => {
     const [session, loading] = useSession();
 
-    // For spacing, when the #cards in a row < max #cards
-    const placeholders: ProgramCardInfo[] = [
-        {
-            id: 0,
-            name: "placeholder",
-            image: "",
-            description: "",
-            startDate: new Date(),
-            endDate: new Date(),
-            onlineFormat: programFormat.online,
-            tag: "",
-            price: 0,
-        },
-        {
-            id: 0,
-            name: "placeholder",
-            image: "",
-            description: "",
-            startDate: new Date(),
-            endDate: new Date(),
-            onlineFormat: programFormat.online,
-            tag: "",
-            price: 0,
-        },
-    ];
+    // const placeholders: ProgramCardInfo[] = [
```

Comment:
Did you mean to delete these?

Reply:
@mahadzaryab1 as well, I wanted to keep in the old code just in case the refactor wasn't what you guys were looking, I'm going to delete it after

Reply:
I think it's fine to delete the code because in the PR, we can see the old code as well.

Reply:
o true, forgot :(